test/Microsoft.AspNetCore.Mvc.ViewFeatures.Test/Internal/PartialViewResultExecutorTest.cs

```diff
@@ -340,7 +340,8 @@ private PartialViewResultExecutor GetViewExecutor(DiagnosticSource diagnosticSou
                 new CompositeViewEngine(options),
                 new TempDataDictionaryFactory(new SessionStateTempDataProvider()),
                 diagnosticSource,
-                NullLoggerFactory.Instance);
+                NullLoggerFactory.Instance,
+                new TestModelMetadataProvider());
```

Comment:
`new TestModelMetadataProvider()` is useful primarily when a test needs to override the metadata i.e. you're about to use `ForProperty()` or `ForType()`. Here, `new EmptyModelMetadataProvider()` matches the rest of the class.